app/views/list/_table.html.erb

```diff
@@ -0,0 +1,21 @@
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header">Name</th>
+      <th class="govuk-table__header">Created At</th>
+      <th class="govuk-table__header">Updated At</th>
+      <th class="govuk-table__header">Service Email</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+        <tr class="govuk-table__row">
+  <% @all_organisations.each do | organisation| %>
+        <td class="govuk-table__cell quarter-column" scope="row"><%=  organisation.name %></td>
+        <td class="govuk-table__cell quarter-column" scope="row"><%=  organisation.created_at.strftime('%e %b %Y %H:%M %P') %></td>
+        <td class="govuk-table__cell quarter-column" scope="row"><%=  organisation.updated_at.strftime('%e %b %Y %H:%M %P') %></td>
+        <td class="govuk-table__cell quarter-column" scope="row"><%=  organisation.service_email %></td>
```

Comment:
Do we want this to be a link like `<a href="mailto:service@email.com>service@email.com</a>` ?

if yes, then this may help: https://apidock.com/rails/ActionView/Helpers/UrlHelper/mail_to

Reply:
Will look into it, not sure how admin wants it to be yet.